client/src/lib/util.ts

```diff
@@ -0,0 +1,20 @@
+export const pluralize = (
+  amount: number,
+  singleText: string,
+  pluralText: string
+) => {
+  if (amount === 1) {
+    return singleText;
+  }
+  return pluralText;
+};
+
+/**
+ * Create an URL query string, dropping any null or undefined values.
+ */
+export const toQueryString = (items: [string, string | null | undefined][]) => {
+  const definedItems = items.filter(([_, value]) => typeof value === "string");
```

Comment:
Do we also want to filter out empty strings? Not sure about that ...

Reply:
J'y ai pensé justement, et je dirais que non. Un page de recherche avec `?q=` dans l'URL c'est assez courant, et souvent interprété comme "aller à la page du formulaire de recherche". Ça se vérifie sur https://gov.uk ou https://duckduckgo.com.

Par contre je ne suis pas sûr que c'est ce qu'on fait ici...